packages/cli/tests/cli.test.ts

```diff
@@ -179,3 +179,60 @@ describe('Contract function call', () => {
     expect(result.txid).toEqual(txid);
   });
 });
+
+describe('Keychain custom derivation path', () => {
+  test('Make keychain using custom derivation path', async () => {
+    const encrypted = 'vim+XrRNSm+SqSn0MyWNEi/e+UK5kX8WGCLE/sevT6srZG+quzpp911sWP0CcvsExCH1M4DgOfOldMitLdkq1b6rApDwtAcOWdAqiaBk37M=';
+    const args = [encrypted];
+    // @ts-ignore
+    inquirer.prompt = jest.fn().mockResolvedValue({derivationPath: "m/44'/5757'/0'/0/0"});
+
+    // Mock TTY
+    process.stdin.isTTY = true;
+    process.env.password = 'supersecret';
+
+    const keyChain = await makeKeychain(testnetNetwork, args);
+    const result = JSON.parse(keyChain);
+
+    expect(result).toEqual({
+      mnemonic: 'vivid oxygen neutral wheat find thumb cigar wheel board kiwi portion business',
+      keyInfo: {
+        privateKey: 'd1124855494c883c5e1df0201be40a835f08ae5fc3a6520224b2239db94a818001',
+        address: 'ST1J28031BYDX19TYXSNDG9Q4HDB2TBDAM921Y7MS',
+        btcAddress: 'mpeSzfUTBba7qzKNcg8ojNm4GAfwmNPX8X',
+        wif: 'L4E7pXmqdm8C8TakpX7YDDmFopaQw32Ak6V5BpRFNDJmo7wjGVqc',
+        index: 0
+      }
+    });
+    // Unmock TTY
+    process.stdin.isTTY = false;
+    process.env.password = undefined;
+  });
+
+  test('Get stacks wallet key custom derivation path', async () => {
+    const encrypted = 'vim+XrRNSm+SqSn0MyWNEi/e+UK5kX8WGCLE/sevT6srZG+quzpp911sWP0CcvsExCH1M4DgOfOldMitLdkq1b6rApDwtAcOWdAqiaBk37M=';
+    const args = [encrypted];
+    // @ts-ignore
+    inquirer.prompt = jest.fn().mockResolvedValue({derivationPath: "m/44'/5757'/0'/0/0"});
```

Comment:
add a test for a custom path like `m/888'/0'/0`

Reply:
Done